src/components/homepage-sections/BrowseByGenre.tsx

```diff
@@ -21,9 +25,15 @@ const GenreSection = ({ genres, genreVinyls }: GenreSectionProps) => {
         {genres.slice(7, 12).map((genre) => {
           const vinyls = genreVinyls[genre.genre];
           const product = vinyls && vinyls.length > 0 ? vinyls[0] : null;
```

Comment:
I think this is always picking the first vinyl in the list (Pink Floyd) which is why they are all the same, we should pick the first vinyl where the genre key matches the genre being mapped, maybe using a .filter method on the vinyls? like `vinyls.filter(vinyl => vinyl.genre === genre.genre)[0]` ?